soc/sensry/ganymed/sy1xx/Kconfig

```diff
@@ -0,0 +1,147 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 sensry.io
+
+config SOC_SERIES_SY1XX
+    bool
+	select RISCV
+    select RISCV_ISA_RV32I
+    select RISCV_ISA_EXT_M
+    select RISCV_ISA_EXT_ZICSR
+    select RISCV_PRIVILEGED
+
+
+if SOC_SERIES_SY1XX
+
+config RISCV_PMP
```

Comment:
these go in the `Kconfig.defconfig` file

Reply:
done